### DISCUSSION TITLE
Valutazione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#183] [Idee e proposte]
Buongiorno,sarebbe possibile lasciare alla segreteria la facoltà di decidere se usare nella valutazione dei docenti  anche i=impreparato,md=materiale dimenticato ,o=ottimo,d=distinto=s=sufficiente oltre ai voti numerici?

**Comment by trinko** [comment]
Ricordo che oltre il voto, o al posto del voto, è sempre possibile aggiungere un giudizio di qualsiasi tipo
